mkdocs.yml

```diff
@@ -77,6 +77,7 @@ nav:
         - Git Repositories: 'models/extras/gitrepository.md'
         - Health Check: 'additional-features/healthcheck.md'
         - Jobs: 'additional-features/jobs.md'
+        - Job Scheduling and Approvals: 'additional-features/job_scheduling_and_approvals.md'
```

Comment:
For consistency, this filename/path should be `job-scheduling-and-approvals.md` rather than `job_scheduling_and_approvals.md`.